### PR TITLE
Add angular form field and button to page.

### PR DESCRIPTION
**Pull request by jagjennings**
- insert navbar that Efrain created in other repo
- create form fields for name and mp3 file
- add button that calls getData() when clicked (does not fetch data yet, reutrns message)

sidebar closed: https://screenshot.googleplex.com/zWm8jHMS6fr.png
sidebar open: https://screenshot.googleplex.com/vebMTK9A8PP.png
button click message: https://screenshot.googleplex.com/jy4mCUWttVS.png

### CODE REVIEW

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.center-page {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css]: Nit: here and elsewhere can we keep the style attributes alphabetized?

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+.sidenav {
+width: 300px;
+overflow: hidden;
+}
+
+.sidenav .mat-toolbar {
+background: inherit;
+}
+.mat-toolbar{
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css:19]: Please keep spacing consistent here and below.

File: launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+@Component({
+  selector: 'app-create-form',
+  templateUrl: './create-form.component.html',
+  styleUrls: ['./create-form.component.css']
+})
+export class CreateFormComponent implements OnInit {
+  constructor() {}
+
+  // fetches input data when user clicks create button
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.ts]: Nit: Please punctuate comments as full sentences! "Fetches input data when user clicks create button."

**Review by bramanudom — COMMENTED** [review]
Great job! I have just a few nits. 

Next time, would it be possible to break down the PR into smaller sizes? Maybe breka down the cli generated boilerplate and send that out as a quick and easy PR, and then follow up with your additions.

**Comment by jagjennings** [comment]
> Great job! I have just a few nits.
> 
> Next time, would it be possible to break down the PR into smaller sizes? Maybe breka down the cli generated boilerplate and send that out as a quick and easy PR, and then follow up with your additions.

**Comment by jagjennings** [comment]
> > Great job! I have just a few nits.
> > Next time, would it be possible to break down the PR into smaller sizes? Maybe breka down the cli generated boilerplate and send that out as a quick and easy PR, and then follow up with your additions.

Will do!

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css (hunk 1 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css]: done.

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css (hunk 2 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.css:19]: done.

File: launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.ts (hunk 3 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.ts]: done.

**Comment by jagjennings** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fstep18-2020%2Fpull%2F3) for more info**.

@googlebot I consent.

**Comment by jagjennings** [comment]
@googlebot I consent.

**Comment by ERetana2** [comment]
@googlebot I consent.

**Review by bramanudom — APPROVED** [review]

**Review by qihuang2 — APPROVED** [review]

**Pull request merged**